object/types.proto

```diff
@@ -87,8 +87,19 @@ message Header {
   // There are some "well-known" attributes starting with `__NEOFS__` prefix
   // that affect system behaviour:
   //
-  // * __NEOFS__UPLOAD_ID
-  // * __NEOFS__EXPIRATION_EPOCH
+  // * __NEOFS__UPLOAD_ID \
+  //   Marks smaller parts of a split bigger object
+  // * __NEOFS__EXPIRATION_EPOCH \
+  //   Tells GC to delete object after that epoch
+  //
+  // And some well-known attributes used by applications only:
+  //
+  // * Name \
+  //   Human-friendly name
+  // * FileName \
+  //   File name to be associated with the object on saving
```

Comment:
Not sure about having both `Name` and `FileName`. What's the difference then?

Reply:
File can be split for multiple objects and each split object can have some unique custom text identifier independent from original file name. Got it.